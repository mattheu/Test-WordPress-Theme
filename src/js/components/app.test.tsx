import React from 'react';
import App from './app';
import TestRenderer from 'react-test-renderer'; // ES6

test( 'Prepends the word "test"', () => {
	const testRenderer = TestRenderer.create(
		<App />
	);

	expect( testRenderer.toJSON() ).toMatchSnapshot();
});
