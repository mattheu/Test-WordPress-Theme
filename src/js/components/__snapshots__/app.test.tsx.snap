// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prepends the word "test" 1`] = `
<div>
  Hello test World
</div>
`;
